Sources/SwiftDocC/Model/Rendering/Symbol/AvailabilityRenderMetadataItem.swift

```diff
@@ -139,6 +139,23 @@ public struct AvailabilityRenderItem: Codable, Hashable, Equatable {
             isBeta = false
         }
     }
+
+    init?(_ availability: MetadataAvailability, current: PlatformVersion?) {
+        if availability.introduced == nil {
+            // FIXME: Deprecated/Beta markings need platform versions to display properly in Swift-DocC-Render (rdar://56897597)
+            // Fill in the appropriate values here when that's fixed (https://github.com/apple/swift-docc/issues/441)
+            return nil
+        }
+
+        let platformName: PlatformName?
+        if availability.platform == .any {
+            platformName = nil
+        } else {
+            platformName = PlatformName(operatingSystemName: availability.platform.rawValue)
+        }
```

Comment:
Isn't this the same code as [this extension][1]?

[1]: https://github.com/apple/swift-docc/pull/440/files#diff-b00949c03295a335ed2ba8e0254f7e534480adae2a1ac1d757a141bf479db214R108

```suggestion
        let platformName = PlatformName(metadataPlatform: availability.platform)
```